### PR TITLE
Newsletter vol.5

### PR DESCRIPTION
**Pull request by syleeeee**
Modifications:

- Added Newsletter vol. 5
- Replaced Slack with Discord
- Replaced StackOverflow with GitHub Discussions
- Terminate virtual office hours
- Minor visual tweaks on the site header
- Reduced the z-index of the star-begging popup so it doesn't hide the cookie consent.

Result:

screencapture-localhost-8000-news-20230414-newsletter-5-2023-04-20-10_51_27

### CODE REVIEW

**Comment by syleeeee** [comment]
Sadly there's no discord icon yet. 
https://github.com/ant-design/ant-design-icons/issues/334

File: site/gatsby-config.js [hunk 1]
```diff
@@ -11,6 +11,10 @@ module.exports = {
         name: 'slack',
         href: 'https://join.slack.com/t/line-armeria/shared_invite/enQtNjIxNDU1ODU1MTI2LWRlMGRjMzIwOTIzMzA2NDA1NGMwMTg2MTA3MzE4MDYyMjUxMjRlNWRiZTc1N2Q3ZGRjNDA5ZDZhZTI1NGEwODk',
       },
+      {
+              name: 'discord',
+              href: 'https://discord.gg/7FH8c6npmg',
+      },
```
Comment (trustin) [inline site/gatsby-config.js:17]:
```suggestion
      {
        name: 'discord',
        href: 'https://discord.gg/7FH8c6npmg',
      },
```

**Comment by trustin** [comment]
Let me fix the Discord icon tomorrow 😉

File: site/src/pages/news/20230414-newsletter-5.mdx [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+  <img src="../../images/armeria.gif" loading="lazy" width="282" height="112" />
+</video>
+
+## Announcements
+
+<div class="noBullet">
+
+We have some announcements to share with you.
+
+- 📢 **Discontinuance of the virtual office hour**:
```
Comment (jrhee17) [inline site/src/pages/news/20230414-newsletter-5.mdx:17]:
```suggestion
- 📢 **Discontinuance of virtual office hours**:
```

File: site/src/pages/news/20230414-newsletter-5.mdx [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+  - We will transform the existing virtual hours to another form such as quarterly webinar. But we haven't decided how often we hold the webinar or what to cover. Please stay tuned.
+- 🙌 **[Armeria adopters page](https://github.com/line/armeria/discussions/4626) and limited goods package**:
+  - You can now show off how well your service take adventage of Armeria!
```
Comment (jrhee17) [inline site/src/pages/news/20230414-newsletter-5.mdx:20]:
```suggestion
  - You can now show off how well your service take advantage of Armeria!
```

File: site/src/pages/news/20230414-newsletter-5.mdx [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+  - Don't forget to request the limited goods packages.
+<img src="../../images/limited_package.jpg" />
+- 🚛 **Moving to Discord**:
+  - We dicided to move to Discord to preserve all the past messages.
```
Comment (jrhee17) [inline site/src/pages/news/20230414-newsletter-5.mdx:24]: "past messages" can be counted

```suggestion
  - We dicided to move to Discord to preserve all past messages.
```

File: site/src/pages/news/20230414-newsletter-5.mdx [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+
+## From the devs
+
+<div class="noBullet">
+
+- ✨ We've released several new versions since our last newsletter with a bunch of new exciting features! Here are the highlights between [1.17.1](https://armeria.dev/release-notes/1.17.1) and [1.23.1](https://armeria.dev/release-notes/1.23.1).
+  - Resilience4j circuit breaker support
+  - Improvements in gRPC support
+    - Asynchronous ServerInterceptor + Kotlin Coroutine support
+    - Annotations support
```
Comment (jrhee17) [inline site/src/pages/news/20230414-newsletter-5.mdx:40]:
```suggestion
    - Improved annotations support
```

File: site/src/pages/news/20230414-newsletter-5.mdx [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+  - #4569 Consider RFC 7807 Problem Details for HTTP APIs as the default response format in REST APIs
+  - #4738 Consider a way to make OAuth2AuthorizaitonGrant dynamic when update the grant information needed
+
+</div>
+
+## Good first issues
+
+<div class="noBullet">
+
+🔰 Want to contribute but not sure where to start from? Try one of these:
```
Comment (jrhee17) [inline site/src/pages/news/20230414-newsletter-5.mdx:84]:
```suggestion
🔰 Want to contribute but not sure where to start from? Try this issue:
```

**Review by jrhee17 — APPROVED** [review]
Looks good! Left some minor comments @syleeeee 🙇

**Comment by syleeeee** [comment]
Thanks all, I checked all comments.

**Review by trustin — APPROVED** [review]
Thanks a lot, @syleeeee ! 🙇

**Review by ikhoon — APPROVED** [review]
🙇‍♂️

**Review by minwoox — APPROVED** [review]
Thanks a lot, @syleeeee! 😄

**Pull request merged**